docs/api_docs/tfra/dynamic_embedding/FileSystemSaveableObjectCreator.md

```diff
@@ -0,0 +1,81 @@
+<div itemscope itemtype="http://developers.google.com/ReferenceObject">
+<meta itemprop="name" content="tfra.dynamic_embedding.FileSystemSaveableObjectCreator" />
+<meta itemprop="property" content="__init__"/>
+</div>
+
+# tfra.dynamic_embedding.FileSystemSaveableObjectCreator
+
+<!-- Insert buttons and diff -->
+
+<table class="tfo-notebook-buttons tfo-api" align="left">
+
+<td>
+  <a target="_blank" href="https://github.com/tensorflow/recommenders-addons/tree/master/tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_creator.py">
+    <img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />
+    View source on GitHub
+  </a>
+</td></table>
+<br/>
+<br/>
+<br/>
+<br/>
+
+
+
+## Class `FileSystemSaveableObjectCreator`
+
+A creator for expanding and shrinking the number of tables in distributed training by independently saving files.
+
+Inherits From: `SaveableObjectCreator`
+
+<!-- Placeholder for "Used in" -->
+
+<h2 id="__init__"><code>__init__</code></h2>
+
+<a target="_blank" href="https://github.com/tensorflow/recommenders-addons/tree/master/tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_creator.py">View source</a>
+
+``` python
+__init__(config)
+```
+
+A creator for dynamic variable is created. 
+Which could be as a parameter saveable_object_creator for get_variable function.
+
+The parameter of FileSystemSaveableObjectCreator initial function is a `FileSystemSaveableObjectConfig` class. And it receives a string initial parameter as the saving path. 
+
+If there is no config parameter for `FileSystemSaveableObjectCreator`, default saving path is `"savedmodel/variables/TFRADynamicEmbedding"`.
+
+#### Args:
+
+
+* <b>`config`</b>: A `tfra.dynamic_embedding.FileSystemSaveableObjectConfig` class for specific saving path.
+
+
+
+## Useage
+``` python
+import tensorflow_recommenders_addons.dynamic_embedding as de
+
+test_devices = ['GPU:0', 'GPU:1']
+params0 = de.get_variable(
+            'table0',
+            devices=test_devices,
+            dim=8,
+            initializer=0.0,
+            bp_v2=True,
+            saveable_object_creator=de.FileSystemSaveableObjectCreator())
```

Comment:
There are not too many comments for me here, I just want to know if we have better choice for the parameter, this requires the user to understand the internal mechanism of `SaveableObject`, it's too professional maybe. If we can change it to `saver` and `saverCreater` and `ShardSaver`?

Reply:
saver_creator?